api/src/main/java/au/org/aodn/nrmn/restapi/service/SurveyContentsHandler.java

```diff
@@ -45,7 +45,7 @@ public void startRow(int rowNum) {
         isDataRow = (rowNum > 1);
         rowHasData = false;
         if (isDataRow) {
-            currentRow = StagedRow.builder().pos(rowNum - 1).build();
+            currentRow = StagedRow.builder().pos((rowNum - 1) * 1000).build();
```

Comment:
Whats this magic number?

Reply:
Some slack so that the client can increment 1 to the row number when cloning a row and still maintain row order.